jekyll/_cci2/github-app-trigger-options.adoc

```diff
@@ -0,0 +1,83 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= GitHub App trigger options
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A guide to the options available for triggering GitHub App pipelines
+:icons: font
+:experimental:
+
+When you create a trigger for a GitHub App pipeline, you can choose from a set of trigger options that correspond to link:https://docs.github.com/en/webhooks/webhook-events-and-payloads[GitHub App events].
+
+== Introduction
+
+When you set up a project in CircleCI you will also set up your first pipeline and trigger. This first "default" trigger is an "all push" trigger, which corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event]. You can also set up additional triggers for your project during setup, or later in your menu:Project settings[Triggers].
+
+Once your project is set up you can create additional pipelines and triggers (menu:Project settings[Triggers]). You can configure your triggers to initiate pipelines on a variety of events, including non-push events, for example, a pull request being opened, or refined push events, for example, only pushes to a default branch, or pushes to a tag.
+
+== Scope of push event support
+
+* Pipelines are not triggered on branch or tag deletion push events, even when you have the "All push" trigger option selected.
+* Pipelines are not triggered when a new branch is created if there are no new commits on the branch.
+
+== Supported trigger events
+
+GitHub App triggers allow you to define the work to do (pipeline) when an event takes place (trigger). GitHub App triggers are an orchestration tool available in CircleCI. You can access GitHub App triggers using the "Run on" menu when setting up a trigger. The following preset trigger options are available:
```

Comment:
I think we need a name of them. I'd go "trigger rules" because that's how we call them in the API. 
```suggestion
GitHub App triggers allow you to define the work to do (pipeline) when an event takes place (trigger). GitHub App triggers are an orchestration tool available in CircleCI. You can access GitHub App trigger rules using the "Run on" menu when setting up a trigger. The following preset trigger options are available:
```